src/flake8_json_reporter/reporters.py

```diff
@@ -46,30 +62,72 @@ def finished(self, filename):
         self.files_reported_count += 1
         self.write_line("]")
 
-    def dictionary_from(self, violation):
```

Comment:
I believe this breaks the CodeClimate one

Reply:
In what way? It has its own `dictionary_from` (doesn't matter though, I made the new formatter inherit from the old one so the method stays where it was)